Gemfile

```diff
@@ -6,4 +6,5 @@ gem 'discordrb'
 group :nerds do
   gem 'pry'
   gem 'rubocop'
+  gem 'rbnacl'
```

Comment:
please move this gem to outside the `:nerds` group, because it's required for the shout plugin to run.